app/services/api/v1/ecf_participants/index.rb

```diff
@@ -74,47 +69,6 @@ def updated_since
         rescue ArgumentError
           Time.iso8601(URI.decode_www_form_component(filter[:updated_since]))
         end
-
-        # inner most query
-        # this query deals with the following scenario
-        # given one profile with 2 induction records
-        # we only want to return the latest induction record
-        # we also exclude any partnerships that have been challenged
-        def induction_record_ids_with_deduped_induction_records
-          query = InductionRecord
-            .joins(induction_programme: { partnership: %i[lead_provider cohort] })
-            .joins(participant_profile: %i[user participant_identity])
-            .select("DISTINCT ON (participant_profiles.id) participant_profile_id, induction_records.id")
-            .where(
-              induction_programme: {
-                partnerships: {
-                  challenged_at: nil,
-                  challenge_reason: nil,
-                  lead_provider:,
-                  cohort: with_cohorts,
-                },
-              },
-            )
-            .order("participant_profiles.id", start_date: :desc)
```

Comment:
my understanding of the purpose of this PR is to select the "correct" induction record for a participant. at the moment it does not correctly select the "correct" induction record. this now seems to be based off the most recently created induction record. could we have not simply changed this line to following to achieve the same result?
```suggestion
            .order("induction_records.created_at DESC")
```